spec/features/proposal/collection_draft_proposals/collection_draft_proposal_list_spec.rb

```diff
@@ -189,12 +189,12 @@
 
   context 'when logged in as an approver' do
     before do
-      login
+      real_login(method: 'urs')
       mock_urs_get_users
```

Comment:
This was removed above.  Can it be removed here?

Reply:
I actually didn't remove it above, it just got moved down to line 11. I tried removing them and the tests failed so we need to keep the calls.